### PR TITLE
Changing Structure for doc, content for ubuntu, general design adjust…

### PR DESCRIPTION
**Pull request by jvollhueter**
…ments

- structure in doc:
Existing content is merged into two documents (installation and
 First Applications) and operating system specific paragraphs are
 inserted which can be opened via a pop-up menu
- content for ubuntu:
Some content in the doc for ubuntu is added
- general design adjustments:
 The logo of the TU and the UFZ are no longer
 inserted at the end of each page, but in the general design
 settings as footer

### CODE REVIEW

**Comment by jbathmann** [comment]
For the english Version: Please add "Documentation" button to the main menu

File: web/content/de/docs/Erste_Schritte/Installation.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+<details>
+<summary background="coral";
+  margin-bottom="1em";>Installation von Python in Ubuntu</summary>
+<p>
+
+Ubuntu 18.04 liefert eine erste Installation von (Python 2 und) Python 3 von Haus aus mit. Um zu überprüfen, welche Version sich aktuell auf dem Rechner befindet, kann, nachdem ein neues Terminal-Fenster mit der Tastenkombination **"STRG + Alt + T"** geöffnet wurde, eine Versionsabfrage mit dem Kommando 
+
+	• python3 -V 
+
+erfolgen. Es wird empfohlen das Paket zunächst zu updaten. Um die Version auf den neuesten Stand zu bringen, kann über die Kommandos
```
Comment (jbathmann) [inline web/content/de/docs/Erste_Schritte/Installation.md:31]:
```suggestion
erfolgen. Es wird empfohlen das Paketverzeichnis des Betriebssystems zunächst zu updaten. Um die Version auf den neuesten Stand zu bringen, kann über die Kommandos 
```

File: web/content/de/docs/Erste_Schritte/Installation.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+<figcaption><font size = "1"><i><b>Abbildung 3:</b> Ausführung der Windows exe von Python 3.7.7.</i></font></figcaption>
+</figure><p>
+
+Damit ist die Installation von **Python** abgeschlossen.
+
+</p>
+</details>
+
+<br>
+Nachdem Sie Python auf Ihrem Rechner eingerichtet haben, wird im nächsten Schritt pyMANGA installiert.
```
Comment (jbathmann) [inline web/content/de/docs/Erste_Schritte/Installation.md:84]:
```suggestion
Nachdem Sie Python auf Ihrem Rechner eingerichtet haben, wird im nächsten Schritt pyMANGA installiert. 
Dies können Sie entweder durch Klonen des git-repositories (Für Fortgeschrittene, LINK) oder durch Befolgen der Nachfolgenden Anleitung erreichen.
```

File: web/content/de/docs/Erste_Schritte/Erste_Anwendungen_von_pyMANGA.md [hunk 3]
```diff
@@ -1,21 +1,38 @@
...
 
 <figure>
 <img src="/pictures/ausgefuehrte_main_py_Datei_in_der_Eingabekonsole.jpg">
-<figcaption><font size = "1"><i><b>Abbildung 1:</b> Ausgeführte main.py Datei in der Eingabekonsole.</i></font></figcaption>
+<figcaption><font size = "1"><i><b>Abbildung 1:</b> Ausgeführte main.py Datei in der Windows-Eingabekonsole.</i></font></figcaption>
 </figure><p>
 
 Hier ist zu sehen, dass die ***main.py*** Datei ausgeführt wurde und auf weitere Eingaben wartet. Damit ist der Start von MANGA geglückt und Sie können ein paar erste Verwendungsbeispiele testen. Dazu können Sie folgenden Code eingeben (siehe Abbildung 2).
 
 	• py main.py -i ProjectLib\ExampleSetups\AllSimple_WithOutput.xml			  [1]
```
Comment (jbathmann) [inline web/content/de/docs/Erste_Schritte/Erste_Anwendungen_von_pyMANGA.md:33]: Please update

File: web/content/de/docs/Erste_Schritte/Erste_Anwendungen_von_pyMANGA.md [hunk 4]
```diff
@@ -54,10 +71,25 @@ Analog dazu können Sie mit den folgenden Codes zwei weitere Beispiel ausprobier
 	• py main.py -i ProjectLib\ExampleSetups\FIXEDSAL_BETTINA.xml 			         [2] 
 	• py main.py -i ProjectLib\ExampleSetups\FON_SAZOI_KIWI.xml			         [3]
```
Comment (jbathmann) [inline web/content/de/docs/Erste_Schritte/Erste_Anwendungen_von_pyMANGA.md:72]: Please update

Comment (jbathmann) [reply web/content/de/docs/Erste_Schritte/Erste_Anwendungen_von_pyMANGA.md:72]: Also: please update references within the text above

File: web/content/de/docs/Erste_Schritte/Erste_Anwendungen_von_pyMANGA.md [hunk 5]
```diff
@@ -54,10 +71,25 @@ Analog dazu können Sie mit den folgenden Codes zwei weitere Beispiel ausprobier
...
+## Komplexere Beispiel-Setups mit OpenGeoSys
+
 
 Die nächste Anwendung von **pyMANGA** nutzt **OpenGeoSys** (OGS). Dabei handelt es sich um ein wissenschaftliches Open-Source-Projekt zur Entwicklung numerischer Methoden für die Simulation von thermo-hydro-mechanisch-chemischen (THMC) Prozessen in porösen und fragmentierten Medien. Um **OGS** zu nutzen müssen Sie dieses zunächst herunterladen und installieren. Dazu gehen Sie auf die folgende Internetseite [**Link**](https://www.opengeosys.org/releases/ "https://www.opengeosys.org/releases/")  und scrollen bis Sie die **Version** **6.3.0** finden und downloaden diese (siehe Abbildung 7 und 8).
 
+
+<details>
+<summary >Erste Anwendungen in Ubuntu</summary>
+<p>
+Hier finden Sie in kürze eine Beschreibung für Ubuntu. Sehen Sie sich solange gerne den bereits vorhandenen Inhalt zu Windows an.
```
Comment (jbathmann) [inline web/content/de/docs/Erste_Schritte/Erste_Anwendungen_von_pyMANGA.md:85]: :D

**Review by jbathmann — COMMENTED** [review]
This is nice. Could you also add the same stuff for the English version?

**Comment by jvollhueter** [comment]
Here already the adjustments of the changes of the current state, the next step will be the translation into English.

**Comment by jbathmann** [comment]
Looks good!

**Pull request merged**